src/logger/logger.ts

```diff
@@ -152,14 +165,9 @@ class Logger {
       records.push({ error: this.convertErr(err) });
     }
 
-    const entry: Record<string, unknown> = { severity };
-    for (const [key, value] of Object.entries(records)) {
-      entry[key] = value;
-    }
```

Comment:
The `records` is an array of `Record<string, unknown>`.
So, these lines were added each index and element of the array to the `entry` object.
As a result, log message contained such `index` keys. ( `"0"` )

That is the problem which I want to fix by this PR.